### PR TITLE
#3969 [Manager] Add eye-catching icons to error event logs

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3969 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/EventLog/EventLogRowStyles.js [hunk 1]
```diff
@@ -29,6 +29,22 @@ export default styled.div(
...
+        transform: rotate(45deg);
+        height: 10px;
+        width: 10px;
+        color: white;
+        background: ${theme.palette.error.main};
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        border-radius: 100%;
+      }
```
Comment (eechih) [inline ohara-manager/client/src/components/EventLog/EventLogRowStyles.js]: Thanks @konekoya for the help 👍

File: ohara-manager/client/src/components/EventLog/EventLogHeader.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ const EventLogHeader = ({ onFilter }) => {
           <QuickSearch
             className="search"
             data={logs}
-            keys={['title']}
+            keys={['title', 'payload.errors[0].message']}
```
Comment (eechih) [inline ohara-manager/client/src/components/EventLog/EventLogHeader.js]: 增加 quick filter 的範圍

File: ohara-manager/client/src/components/EventLog/EventLog.js [hunk 3]
```diff
@@ -44,7 +44,15 @@ const EventLog = () => {
...
     const count = size(logs);
-    return count > 0 ? `Total ${count} logs` : 'No log';
+    if (!count) return 'No log';
+
+    const limit = get(settings, 'data.limit');
+    const unlimited = get(settings, 'data.unlimited');
+    if (unlimited) {
+      return `There are ${count} currently`;
+    } else {
+      return `There are ${count} currently (up to ${limit})`;
```
Comment (eechih) [inline ohara-manager/client/src/components/EventLog/EventLog.js]: Display the maximum number of logs

**Comment by eechih** [comment]
retry manager-api

**Comment by saivirtue** [comment]
The failed test was traced in #3940, you can ignore it.

**Pull request merged**